.github/workflows/build_website.yml

```diff
@@ -0,0 +1,32 @@
+name: Build website
+
+on: [push, pull_request]
+
+jobs:
+  build_website:
+    name: "Build website"
+    runs-on: 'ubuntu-latest'
+    steps:
+    - name: Checkout kivy-website
+      uses: actions/checkout@v2
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Set up node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - name: Install python dependencies via requirements.txt
+      run: pip install -r requirements.txt
+    - name: Install tailwindcss via npm
+      run: npm install -D tailwindcss
+    - name: Build (minified) main.css via tailwindcss
+      run: npx tailwindcss -i ./content/static/css/new-kivy.css -o ./content/static/css/main.css --minify
```

Comment:
This could be part of a package.json (and package-lock.json) file, that way we can have these commands in the `scripts` section, plus have `tailwindcss` and dependencies pinned to last known working version

Reply:
Oh. Absolutely.
- Added a script section
- we're now tracking `package.json` and `package-lock.json`
- Updated workflow
- Updated README.md